fhir-operation-bulkdata/pom.xml

```diff
@@ -34,5 +34,10 @@
             <artifactId>jsonassert</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-server</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
I think this should be removed.

Reply:
this is added for being able to use the util function there to read config from server.xml

Reply:
changed to use fhir server config.

Reply:
pom.xml needs to be updated - remove fhir-server

Reply:
it's removed from the pom